test/extended/quota/quota.go

```diff
@@ -0,0 +1,80 @@
+package quota
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area/quota]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-quota")
+	)
+
+	g.It("Author:zxiao-Critical-[API] Verify that quota works for cluster-scoped resources across all namespaces in the workspace", func() {
+		g.By("# Create test workspace")
+		k.SetupWorkSpace()
+
+		g.By("# List secrets, config-map resources")
+		err := k.Run("get").Args("cm").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = k.Run("get").Args("secrets").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("# Create quota under the current workspace")
```

Comment:
Currently, only support creating quota in a namespace called admin. Detail see https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-53980, something wrong with counting worksapce, detail see https://coreos.slack.com/archives/C020QAF9D46/p1664419562486969?thread_ts=1663600223.134799&cid=C020QAF9D46